packages/web3/package.json

```diff
@@ -30,16 +30,18 @@
 		"test:ci": "jest --coverage=true --coverage-reporters=json --verbose",
 		"test:watch": "npm test -- --watch",
 		"test:unit": "jest --config=./test/unit/jest.config.js",
-		"test:integration": "jest --config=./test/integration/jest.config.js --runInBand --passWithNoTests"
+		"test:integration": "jest --config=./test/integration/jest.config.js --runInBand --forceExit"
```

Comment:
Try if you can avoid using `--forceExit` by properly closing the socket connections. This options sometimes hides floating promises.